src/plugins/kibana/public/dashboard/services/saved_dashboards.js

```diff
@@ -69,7 +69,7 @@ module.service('savedDashboards', function (Promise, SavedDashboard, kbnIndex, e
         query: {
           simple_query_string: {
             query: searchString + '*',
-            fields: ['title^3', 'description'],
+            stored_fields: ['title^3', 'description'],
```

Comment:
Seems dubious - we search indexed fields not stored ones so either way is odd naming.